nginx-proxy.conf

```diff
@@ -1,9 +1,10 @@
 include /etc/nginx/api-proxy.conf;
 
 location / {
-    rewrite (?i)\.(jp(e)?g|gif|png|ico|mp(3|4)|webm|og(a|g|m|v|x)|spx|opus|pdf|ttf|tar|gz|tgz|bz2|tbz2|zip)$ https://static.zooniverse.org/$host$request_uri;
+    # TODO: Long term ensure this eventually points to `static.zooniverse.org` CDN domain for CDN benefits
+    rewrite (?i)\.(jp(e)?g|gif|png|ico|mp(3|4)|webm|og(a|g|m|v|x)|spx|opus|pdf|ttf|tar|gz|tgz|bz2|tbz2|zip)$ https://zooniversestatic.z13.web.core.windows.net/$host$request_uri;
```

Comment:
We don't want this to switch to being a non-zooniverse domain, see #222 

What's the reasoning behind not switching the static.zooniverse.org to proxy to azure instead of s3?

Reply:
same reasoning as the last bunch of times this same thing was done, to avoid cutting over everything else at the same time.